src/frontend/web_application/src/components/ContactDetails/components/OrgaForm/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../../Icon';
+import Button from '../../../Button';
+import { FieldErrors, Fieldset, Legend, TextFieldGroup, FormGrid, FormRow, FormColumn } from '../../../form';
+
+import './style.scss';
+
+
+class OrgaForm extends Component {
+  static propTypes = {
+    errors: PropTypes.arrayOf(PropTypes.string),
+    onSubmit: PropTypes.func.isRequired,
+    __: PropTypes.func.isRequired,
```

Comment:
@iamdey don't you think the purpose of OrgaForm is to add a new organization, so the original organization doesn't have to appear here? Original organizations are returned in OrgaDetails, with remove Button on editMode (just like other forms on the page)